datadog/resource_datadog_logs_archive_order.go

```diff
@@ -0,0 +1,91 @@
+package datadog
+
+import (
+	"fmt"
+	datadogV2 "github.com/DataDog/datadog-api-client-go/api/v2/datadog"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"strings"
+)
+
+func resourceDatadogLogsArchiveOrder() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceDatadogLogsArchiveOrderCreate,
+		Update: resourceDatadogLogsArchiveOrderUpdate,
+		Read:   resourceDatadogLogsArchiveOrderRead,
+		Delete: resourceDatadogLogsArchiveOrderDelete,
+		Exists: resourceDatadogLogsArchiveOrderExists,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+		Schema: map[string]*schema.Schema{
+			"name": {Type: schema.TypeString, Required: true},
```

Comment:
Do we really need to have this for the user? It looks like it's an internal field.